.github/workflows/dockerimage.yml

```diff
@@ -14,8 +14,11 @@ jobs:
       fail-fast: false
       matrix:
         php_version: ["5.6", "7.0", "7.1", "7.2", "7.3"]
+        php_pcov_version: ["7.2-pcov", "7.3-pcov"]
```

Comment:
Should just go in the `php_version` array, otherwise it will generate a matrix of every combination possible between `php_version` and `php_pcov_version`